### PR TITLE
Add new snapshot content

### PR DESCRIPTION
**Pull request by chensixx**
for newly created documents, metadata has new properties for container creation stats: createContainerRuntimeVersion and createContainerTimestamp etc. (https://github.com/microsoft/FluidFramework/pull/7527) 
so adding new snapshot that reflect this change

### CODE REVIEW

File: snapshotTestContent/createContainerMetadata/current_snapshots/snapshot_72_0.json [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+                                "path": "content",
+                                "value": {
+                                  "entries": [
+                                    {
+                                      "path": "header",
+                                      "mode": "100644",
+                                      "type": "Blob",
+                                      "value": {
+                                        "contents": "{\"chunkStartSegmentIndex\":0,\"chunkSegmentCount\":2,\"chunkLengthChars\":13,\"totalLengthChars\":13,\"totalSegmentCount\":2,\"chunkSequenceNumber\":72,\"segmentTexts\":[{\"text\":\"new document\",\"props\":{\"attribution\":{\"id\":\"chensi@microsoft.com\",\"name\":\"Si Chen\",\"email\":\"chensi@microsoft.com\",\"timestamp\":1641336000000}}},{\"marker\":{\"refType\":1},\"props\":{\"markerId\":\"1925c045-d6e5-4161-86b2-16201ee4cf7e\",\"nodeType\":\"Paragraph\",\"list!reference\":\"0;list!list-dbaa67b7-31f2-4fbf-b3a7-6c0b484c54060;0\",\"placeholder!attributeText\":\"Add item\"}}],\"headerMetadata\":{\"orderedChunkMetadata\":[{\"id\":\"header\"}],\"sequenceNumber\":72,\"totalLength\":13,\"totalSegmentCount\":2}}",
```
Comment (anthony-murphy) [inline snapshotTestContent/createContainerMetadata/current_snapshots/snapshot_72_0.json:191]: the context should be anonymized before it add

**Comment by agarwal-navin** [comment]
Did you verify that the tests are running on these new snapshots and they are passing? It might be helpful to create a PR in the main FluidFramework repo that point to this branch for now to validate that everything is working as expected. When this PR is merged, you can update that PR to point to the commit in main.

**Comment by chensixx** [comment]
i ran them locally and they passed, i will also create a PR

**Comment by chensixx** [comment]
this https://github.com/microsoft/FluidFramework/pull/8647 confirms the tests are passing

**Review by agarwal-navin — APPROVED** [review]

**Pull request merged**